### PR TITLE
Web Scraper to Extract COVID-19 Information as per (Issue #31)

### PR DESCRIPTION
**Pull request by tauseefmohammed2**
# Description

Python Web Scraper which extracts COVID Data for Top 10 Affected Countries

Fixes #31 

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X] I have named my files and folder, according to this project's guidelines.
- [X] My code follows the style guidelines of this project.
- [X] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [X] I have commented on my code, particularly in hard-to-understand areas.
- [X] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [X] I have included a requirements.txt file (if external libraries are required.)
- [X] My changes do not produce any warnings.

### CODE REVIEW

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Awesome job @tauseefmohammed2  

There are some changes that u can make:

Suggested changes:
1. You can run chrome in headless mode 
2. You can store the result in CSV file

Mandatory change:

Please add this line  --- **wd.quit()** at the end of your program

**Comment by tauseefmohammed2** [comment]
Thanks for the Feedback!, I will work on the suggested changes.
I totally forgot to add wd.quit() at the end, will change that as well.

**Comment by tauseefmohammed2** [comment]
Added the Changes as Requested.

**Review by chavarera — CHANGES_REQUESTED** [review]

File: Scripts/Web_Scrappers/COVID-19_Scraper/COVIDWebScraper.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+deaths_list = []
+continent_list = []
+table =  wd.find_element_by_id("table3")
+count = 0
+for row in table.find_elements_by_xpath(".//tr"):
+    if count == 0:
+        count += 1
+        continue
+    lst = [td.text for td in row.find_elements_by_xpath(".//td")]
+    country_list.append(lst[0])
```
[the remark this answers is not in the document]
Comment (tauseefmohammed2) [reply Scripts/Web_Scrappers/COVID-19_Scraper/COVIDWebScraper.py:58]: The list that is created from HTML Table **(lst)** has elements in the following form [Country Name, Total Cases, Total Deaths, Continent of the Country], so I created a list for each element **(country_list, cases_list, deaths_list, continents_list)**. That is why I am using 4 append statements. 

.extend() function will append the whole list, but I wanted to store them separately so I am using 4 append statements.

**Review by chavarera — APPROVED** [review]

**Review by tusharnankani — APPROVED** [review]

**Comment by AdityaJ7** [comment]
@all-contributors please add @tauseefmohammed2  for code

**Pull request merged**